### PR TITLE
Save Permissions

### PR DESCRIPTION
**Pull request by mattgrill**

### CODE REVIEW

File: src/components/05_pages/Permissions/Permissions.js [hunk 1]
```diff
@@ -137,6 +137,29 @@ const Permissions = class Permissions extends Component {
...
+              credentials: 'include',
+              headers: {
+                'content-type': 'application/json',
+              },
+              method: 'PATCH',
+            },
+          ),
+        ),
+    ).then(() => {
+      // do a thing.
```
Comment (dawehner) [inline src/components/05_pages/Permissions/Permissions.js]: One day I guess we could display a message?

File: src/components/05_pages/Permissions/Permissions.js [hunk 2]
```diff
@@ -137,6 +137,29 @@ const Permissions = class Permissions extends Component {
       ),
     }));
   };
+  saveRoles = () => {
+    Promise.all(
+      this.state.roles
+        .filter(role => this.state.changedRoles.includes(role.attributes.id))
```
[the remark this answers is not in the document]
Comment (mattgrill) [reply src/components/05_pages/Permissions/Permissions.js]: @dawehner I was very appreciative of this.

**Comment by dawehner** [comment]
merged now

**Comment by dawehner** [comment]
merged now

**Pull request merged**